src/core/internal/string.d

```diff
@@ -28,7 +28,7 @@ Params:
 Returns:
     The unsigned integer value as a string of characters
 */
-char[] unsignedToTempString()(ulong value, return scope char[] buf, uint radix = 10) @safe
+char[] unsignedToTempString(uint radix = 10)(ulong value, return scope char[] buf) @safe
 {
     if (radix < 2)
```

Comment:
You need static if/else here, or else it will complain about unreachable code (in the case where the radix is less than 2, which should never happen)

Alternatively, just statically disallow it. In fact, we should probably disallow greater than base 36 as well.

Reply:
done